.github/workflows/benchmark-action.yml

```diff
@@ -26,9 +26,9 @@ jobs:
       - uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3.6.0
       - uses: actions/setup-dotnet@v4
         with:
-          dotnet-version: '6.0.418'
+          dotnet-version: '8.0.x'
       - name: Run benchmark
-        run: cd tests/Microsoft.Identity.Test.Performance && dotnet run -c release -f net6.0 --exporters json 
+        run: cd tests/Microsoft.Identity.Test.Performance && dotnet run -c release -f net8.0 --exporters json 
```

Comment:
Why don't we delete it instead? Are we using this?

Reply:
afaik we are using it, tagging @pmaytak to confirm

Reply:
I remember we discussed this very thing. As I understand it, We are comparing the performance of the PRs to a baseline that is acquired regularly. (Daily I believe).

Reply:
I see it as disabled now, when I checked it. 

![image](https://github.com/user-attachments/assets/0e59f5c6-0cc4-49d4-8baa-29f7f08b7b7b)

@bgavrilMS is the suggestion to delete? we don't benchmark anymore?

Reply:
@bgavrilMS can you please suggest

Reply:
I think our agreement was that while this approach is not super accurate it can be used to show any abnormal changes in performance; so it still has value. This was supposed to be an intermediate step before a more permanent solution.

Reply:
Thanks for confirming this @pmaytak. It makes sense merge this and turn on the action @bgavrilMS